packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js

```diff
@@ -0,0 +1,11 @@
+export const NO_OS_INSTALLED_REGEX = /^none_\d{2}?$/;
+export const TRAVAUX_LINK = 'http://travaux.ovh.com/';
```

Comment:
Url has changed

Reply:
https://www.status-ovhcloud.com/
This is the new URL?

Reply:
Yes it is!

Reply:
Yes it is!